### PR TITLE
OPSEXP-1688: Update docker compose to ACS 7.2.1 and 7.3.0-M2

### PR DESCRIPTION
**Pull request by piraartur**
OPSEXP-1688: Updating docker compose files and markdown table in readme.md

### CODE REVIEW

File: docker-compose/docker-compose.yml [hunk 1]
```diff
@@ -12,7 +12,7 @@ version: "2"
 
 services:
   alfresco:
-    image: quay.io/alfresco/alfresco-content-repository:7.3.0-M1
+    image: quay.io/alfresco/alfresco-content-repository:7.3.0-A12
```
Comment (gionn) [inline docker-compose/docker-compose.yml:15]: stick to -M versions, -A are alphas

File: docker-compose/7.1.N-docker-compose.yml [hunk 2]
```diff
@@ -12,7 +12,7 @@ version: "2"
 
 services:
   alfresco:
-    image: quay.io/alfresco/alfresco-content-repository:7.1.1.3
+    image: quay.io/alfresco/alfresco-content-repository:7.1.2-8269
```
Comment (gionn) [inline docker-compose/7.1.N-docker-compose.yml:15]: not sure which is the versioning scheme with the old 7.1 but I would say `-dddd` are dev versions, so latest available should be:

```suggestion
    image: quay.io/alfresco/alfresco-content-repository:7.1.1.5
```

**Review by gionn — CHANGES_REQUESTED** [review]

File: docker-compose/docker-compose.yml (hunk 1 above)
Comment (alxgomz) [reply docker-compose/docker-compose.yml:15]: also `share` needs to be updated to match repo image

**Review by alxgomz — CHANGES_REQUESTED** [review]

**Review by gionn — APPROVED** [review]

**Review by alxgomz — APPROVED** [review]
LGTM

**Pull request merged**